README.md

```diff
@@ -166,23 +93,23 @@ The JSON files are produced by the tool shown in this repository. Hence, it is e
 
 ```
 $ python src/main.py -h
-main.py [-h] [-p LOCATION] [-d DATE] [-j PATH] [-c] [--openmensa PATH]
+main.py [-h] [-p CANTEEN] [-d DATE] [-j PATH] [-c] [--openmensa PATH]
         [-l]
 
 optional arguments:
   -h, --help            show this help message and exit
-  -p LOCATION, --parse LOCATION
-                        the location you want to eat at
+  -p CANTEEN, --parse CANTEEN
+                        the canteen you want to eat at
   -d DATE, --date DATE  date (DD.MM.YYYY) of the day of which you want to get
                         the menu
   -j PATH, --jsonify PATH
                         directory for JSON output (date parameter will be
                         ignored if this argument is used)
   -c, --combine         creates a "combined.json" file containing all dishes
-                        for the location specified
+                        for the canteen specified
   --openmensa PATH      directory for OpenMensa XML output (date parameter
                         will be ignored if this argument is used)
-  -l, --locations       prints all available locations formated as JSON
+  -l, --canteens       prints all available canteens formated as JSON
```

Comment:
That’s no good interface. But I do not know how we could call it better since we have the clash with the `—combine` option.

Reply:
just remove the abbrevation, I guess

Reply:
That’s actually a really good idea 👍🏻